package.json

```diff
@@ -27,7 +27,7 @@
     "discord.rss": "5.0.6",
     "dotenv": "8.2.0",
     "erlpack": "github:discordapp/erlpack",
-    "isolated-vm": "3.3.3",
+    "isolated-vm": "^3.3.7",
```

Comment:
Dlaczego?

Reply:
Z tego co pisał @jundymek to poprzednia wersja mu wywalała na localu

Reply:
Wywalało mi błędy przy instalacji modułów. Nie mogłem tego lokalnie ustawić, bo jak `3.3.3` instalowało to się wywalało. Gdzieś tam doczytałem, że był jakiś błąd w tej wersji, a ostatnia to 3.3.7. Zainstalowałem i działało.

Reply:
To tylko wywal `^` i będzie git majonez